lsdcGui.py

```diff
@@ -5166,8 +5166,9 @@ def initUI(self):
         self.immediate_comm_pv = PV(daq_utils.beamlineComm + "immediate_command_s")
         self.stillModeStatePV = PV(daq_utils.pvLookupDict["stillModeStatus"])        
         self.progressDialog = QtWidgets.QProgressDialog()
-        self.progressDialog.setCancelButtonText(QString())
+        self.progressDialog.setCancelButtonText(QString("Cancel"))
```

Comment:
ideally, could you make this a separate commit next time? also, was this done so you could help diagnose what it was?

Reply:
Correct, It was done to verify I was modifying the correct dialog box. I will put something like this in a separate commit next time